contentful/apps
# Hacktag: Add Angular template to /examples

## Purpose

We've gotten feedback from prospective clients that more support for Angular would help with adoption of the Contentful system. Adding an angular example is a path to this. 

## Approach



## Testing steps

- cd into /examples/angular
- npm install
- ng serve
- verify the example serves successfully
